sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/Error.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.digitaltwins.core.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/** Error definition. */
+@Fluent
+public final class Error {
```

Comment:
This was a bit off a miss in the swagger. "Error" and "ErrorResponse", etc. are standard Azure Core SDK models returned with error responses, so using the same names will lead to a collision in practice (i.e., when using the SDK). We saw the same issue in .NET and explicitly renamed this class (to something like "ErrorInformation"). In fact, the validators in .NET explicitly warned us of this and failed compilation. I would expect the same to happen in Java at some point soon. You can still convert to the one generated by autorest, but for the models returned by the client, it would be great to differentiate.